### DISCUSSION TITLE
Why doesn't payout move to "In Progress" after broadcasting it or "Completed" after settled?

### DISCUSSION THREAD

**Discussion by olimits7** [btcpayserver/btcpayserver#4110] [Technical Support] [answered]
Hi,

I created an invoice refund and then broadcasted this payout successfully but it still shows under "**Awaiting Payment**" instead of moving over to "**In Progress**".  And then when refund gets settled it's not moving over to "**Completed**" status.  Shouldn't this happen automatically within **BTCPay Server**?

https://docs.btcpayserver.org/Payouts/

Thank you!

**Comment by olimits7** [comment]
I had **Voltage** upgrade **BTCPay Server** to **v1.6.10.0** and I did another test but didn't work.  After broadcasting payout it still stays under "**Awaiting Payment**" status instead of moving to "**In Progress**" and doesn't move to "**Completed**" status once settled.

**Comment by Kukks** [comment, marked as answer]
I've attempted to replicate but had no success. 

* Did you pay using the btcpay wallet send page?
* Did you by any chance use "Subtract fees from amount" when sending?
* Have you tried refreshing the payouts page after sending? It could take a few seconds for it to update the state of the payout.
* Are you sure the payout was in "awaiting payment" state?
* Are you sure the amount sent matched the amount required exactly?

**Reply by olimits7** [reply]
- Yes, I used btcpay wallet send page.
- Yes, I used "Subtract fees from amount" and also "Always include non-witness UTXO if available" since this is required so it works properly with my Trezor device.
- Yes, currently refund has "10 confirmations" and when I refresh page it still shows under "Awaiting Payment" status.
- I always select the first option in the list on the issue refund pop-up box, "The crypto currency price, at the rate the invoice got paid.".

Please see link below for screenshots of me going through each of the steps to submit this refund.

https://drive.google.com/drive/folders/1IPy9TNdd5eqGpbjImCPU7iwHiSsnPsQd?usp=sharing

Thank you!

**Reply by olimits7** [reply]
**Update:** I was able to test this more and it does work properly when I don't check "**Subtract fees from amount**".  It didn't let me submit before because I didn't have a high enough balance in my BTCPay Server, but it seems to be working properly now. Ty.
